server/src/internalClusterTest/java/org/elasticsearch/index/query/plugin/CustomQueryParserIT.java

```diff
@@ -10,6 +10,8 @@
 
 import org.elasticsearch.index.query.BoolQueryBuilder;
 import org.elasticsearch.plugins.Plugin;
+import org.elasticsearch.search.DummyQueryBuilder;
+import org.elasticsearch.search.DummyQueryParserPlugin;
```

Comment:
this could be reverted?

Reply:
I had to move the DummyQueryBuilder and plugin to :test:framework so I can use it in the mutache module for the template tests. There is no "o.e.index/query/plugin" package there, so I put it next to the MockSearchService et al, this is why the imports are here.

Reply:
got it, it was confusing that only the imports were added here but no usages. Of course it's because you moved the class, makes sense!